src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.data.tag;
+
+import java.util.List;
+
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+	private	final boolean isAdded;
+	private final Person person; 
+	private final Tag tag;
+	
+	public Tagging (boolean isAdded, Person person, Tag tag) {
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.